lib/modules/manager/regex/utils.ts

```diff
@@ -97,3 +98,14 @@ export function mergeExtractionTemplate(
     replaceString: addition.replaceString ?? base.replaceString,
   };
 }
+
+export function isValidDependency({
+  depName,
+  currentValue,
+}: PackageDependency): boolean {
+  // check if all the fields are set
+  return (
+    is.nonEmptyStringAndNotWhitespace(depName) &&
+    is.nonEmptyStringAndNotWhitespace(currentValue)
```

Comment:
i think we should check for `currentValue` or `currentDigest`. for some datasource the current value is optional.